### PR TITLE
adding openedx.yaml

### PR DESCRIPTION
**Pull request by mduboseedx**
This adds an openedx.yaml file so that our bot can properly adds github labels and process the transitions for OSPRs in Jira.

### CODE REVIEW

File: openedx.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
+
+nick: edx_openid
```
Comment (jmbowman) [inline openedx.yaml]: The spec for these files recommends a max of 4 characters for this; maybe just `opid`?

File: openedx.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# This file describes this Open edX repo, as described in OEP-2:
+# http://open-edx-proposals.readthedocs.io/en/latest/oeps/oep-0002.html#specification
```
Comment (jmbowman) [inline openedx.yaml]: The file was renamed, so the current URL for this should be http://open-edx-proposals.readthedocs.io/en/latest/oep-0002-bp-repo-metadata.html#specification

**Comment by mduboseedx** [comment]
@jmbowman Thanks for taking a look. I've made the changes.

**Comment by nedbat** [comment]
This shouldn't have the `openedx-release: {ref: release}` line in it, since it isn't tagged as part of a release (and there is no "release" ref in this repo).

**Pull request merged**